### ISSUE TITLE
cannot support Chinese

### ISSUE THREAD

**Issue by pottergxy** [ubuntu/app-center#1493]
Hi, I am using Raspberry 5 with ubuntu 23.10. When I am using the newest app center, it seems that Chinese cannot be supported. Will be appreciated if you could solve this issue.

**Comment by m-shibata** [comment]
Japanese fonts are not rendered correctly on Ubuntu 24.04 LTS + snap-store 0+git.97116aa either.

It appears to be an issue specific to arm64, as it displays correctly on the amd64 architecture.

```
$ snap list
Name                       Version          Rev    Tracking         Publisher    Notes
bare                       1.0              5      latest/stable    canonical**  base
core22                     20240408         1383   latest/stable    canonical**  base
firefox                    125.0.3-1        4208   latest/stable/…  mozilla**    -
gnome-42-2204              0+git.510a601    178    latest/stable/…  canonical**  -
gtk-common-themes          0.1-81-g442e511  1535   latest/stable/…  canonical**  -
snap-store                 0+git.97116aa    1135   latest/stable/…  canonical**  -
snapd                      2.62             21467  latest/stable    canonical**  snapd
snapd-desktop-integration  0.9              159    latest/stable/…  canonical**  -
thunderbird                115.10.1-1       471    latest/stable    canonical**  -
```

![Screenshot from 2024-05-14 17-22-16](https://github.com/ubuntu/app-center/assets/998856/50338145-e267-4ec6-90bc-84367c034d59)

![Screenshot from 2024-05-14 17-23-13](https://github.com/ubuntu/app-center/assets/998856/2144c2e0-f18b-46f8-a133-3bc59562f1a2)

I also tried the following command, but the result was the same.

```
rm -rf ~/.cache/fontconfig && sudo fc-cache -r -v
```

**Comment by 13142141** [comment]
still have this problem with raspberry pi 5 Ubuntu 24.04 LTS aarch64 when using Simplified Chinese
![截图 2024-07-18 14-58-31](https://github.com/user-attachments/assets/ea575029-cd89-497b-a81a-2691854a5810)

**Comment by d-loose** [comment]
Duplicate of #1658
